packages/types/src/srml/types.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright 2017-2019 @polkadot/types authors & contributors
```

Comment:
New, if you don't care where it comes from, you can just import the types from `@polkadot/types/srml/types`